src/Microsoft.FeatureManagement.AspNetCore/Microsoft.FeatureManagement.AspNetCore.csproj

```diff
@@ -11,7 +11,7 @@
   <Import Project="..\..\build\Versioning.props" />
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
+    <TargetFrameworks>netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
```

Comment:
Please drop net5.0 too. It's a short-term supported version and is already out of support.

Reply:
netcoreapp3.1 is also out of support.

Reply:
Do we want to just fully update this to the current in-support frameworks?

Reply:
Yes.